python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_condition.py

```diff
@@ -49,6 +49,37 @@
 
 
 class AutomationCondition(ABC):
+    """An AutomationCondition represents a condition of an asset that impacts whether it should be
+    executed at a given point in time.
+
+    .. code-block:: python
+
+        from dagster import AutomationCondition, asset
+
+        @asset(automation_condition=AutomationCondition.on_cron("0 0 * * *"))
+        def my_asset(): ...
+
+    AutomationConditions may be combined together into complex expressions using a variety of
```

Comment:
I would use this example as an opportunity to explain what "given point in time" actually means.  Without an example it doesn't really sink in.